apps/web-crawl-q-and-a/web-qa.ipynb

```diff
@@ -627,9 +627,12 @@
     "        # Get the next URL from the queue\n",
     "        url = queue.pop()\n",
     "        print(url) # for debugging and to see the progress\n",
+    "        \n",
+    "        sanitized_url = re.sub(r'[^\\w\\s.-]', '_', url)\n",
```

Comment:
```suggestion
    "        sanitized_url = re.sub(r'[^\w\s.-]', '_', url)\n",
```

are the double backslashes necessary?

Reply:
Thank You for pointing this out. 
I rechecked, and the double slash was renaming everything in the text folder to something obscure although it was working.
I am committing to your suggestion.
Thank You.